bin/wasabi.sh

```diff
@@ -213,12 +222,14 @@ package() {
     done; \
     sed -i '' -e "s|http://localhost:8080|${server}|g" target/constants.json 2>/dev/null; \
     sed -i '' -e "s|VERSIONLOC|${version}|g" target/app/index.html 2>/dev/null; \
+    if [ -z "$OS" ]; then \
     (cd target; \
       npm install; \
       bower install; \
       grunt clean; \
       grunt build --target=develop --no-color); \
```

Comment:
@longdogz in the context of jenkins, we still need to exec the above grunt build, no?

the objective here is to leverage ./bin/wasabi.sh to the fullest extent possible, which works quite well on osx but for linux we will assume (for now) the dependent toolchain we brew install today (eg vagrant, fpm, grunt, et al) is pre-installed, as it is in our internal system. as such, i suspect we need to exec the above non-install operations.